helm/kube-prometheus/charts/exporter-kube-api/templates/service.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: {{ template "name" . }}
+    component: kube-api
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+  name: {{ template "fullname" . }}
+  namespace: kube-system
+spec:
+  clusterIP: None
+  ports:
+    - name: https-metrics
+      port: 443
+      protocol: TCP
+      targetPort: 443
```

Comment:
I think that's fair to hold off, but an example is kargo. We default the kube apiserver port to 6443 so 443 will be available for people wanting to run ingress controllers on 443. 

https://github.com/kubernetes-incubator/kargo/blob/master/roles/kargo-defaults/defaults/main.yaml#L91